packages/i18n/fr.json

```diff
@@ -302,7 +308,15 @@
       "confirmation": "Confirmation",
       "messageResendVerifySignup": "Êtes-vous sûr de vouloir renvoyer un e-mail pour l'inscription de cette personne : {username}?",
       "messageBlockingUser": "Êtes-vous sûr de vouloir bloquer cette personne : {username}?",
-      "noUserFound": "Aucune personne trouvée"
+      "groups": "Groupes",
+      "groupsAvailable": "Groupes disponible",
+      "groupsJoined": "Groupes de la personne",
+      "searchPlaceholder": "Chercher un groupe",
+      "explanationAddingGroups": "Pour attribuer un ou plusieurs groupes à cette personne, sélectionnez les groupes dans la colonne de gauche et cliquez sur le bouton avec la flèche vers la droite.",
+      "explanationRemovingGroups": "Il est également possible de supprimer un ou plusieurs groupes pour cette personne en sélectionnant les groupes dans la colonne de droite.",
+      "noUserFound": "Aucune personne trouvée",
```

Comment:
Well, in fact, I don't know if we need to translate user in `personne` or `utilisateur⋅rice`.
But I think we need to be consistent ?